corehq/form_processor/management/commands/fix_edit_forms_attachments.py

```diff
@@ -105,25 +106,49 @@ def _find_couch_forms_with_missing_attachments():
         return add_forms_with_attachments
 
     @staticmethod
-    def _add_attachment_to_form(attachment_id, form_id):
+    def _add_attachment_to_couch_form(attachment_id, form_id):
         pass
 
+    @staticmethod
+    def _add_attachment_to_sql_form(from_form_id, form_id, attachment_id):
+        src_form = FormAccessorSQL.get_form(from_form_id)
+        attachment = None
+        for attachment_ in src_form.attachments.values():
```

Comment:
This lookup loop could be eliminated if attachment `name` was used instead of `attachment_id`. Getting the attachment would then be as simple as `src_form.attachments[name]`

Reply:
code updated to avoid attachment fetch